### PR TITLE
include FAQ in SUMMARY.md

### PR DESCRIPTION
**Pull request by trunkafunk**
include link to FAQ in gitbook

### Motivation
include and fillout the FAQ section for FS

### CODE REVIEW

File: docs/FAQ.md [hunk 1]
```diff
@@ -6,4 +6,40 @@ description: Frequently Asked Questions
...
 
-The atomic unit for MOB is picoMOB, which is 1e-12. You need u64 to represent MOB, and many frameworks, DBs, and languages top out at u32 or i64. This is why Full-Service json responses are all strings. For i64 issues there is technically no loss of precision, but you need to cast back to u64 when fetching data.
\ No newline at end of file
+The atomic unit for MOB is picoMOB, which is 1e-12. You need u64 to represent MOB, and many frameworks, DBs, and languages top out at u32 or i64. This is why Full-Service json responses are all strings. For i64 issues there is technically no loss of precision, but you need to cast back to u64 when fetching data.
+
+
+
+## How can I make importing of accounts faster?
+When calling [`import_account`](https://mobilecoin.gitbook.io/full-service-api/api-endpoints/v2/account/account/import_account), scanning generally starts at the Origin block (#1).
+By using the **first_block_index** param, a user can specify the starting block for that entropy if it is known. Newly created accounts are given a starting block index.
```
Comment (brucek) [inline docs/FAQ.md:15]: Maybe add something like:

Any transactions that occurred before the **first_block_index** will not be tracked, potentially leading to invalid balances or denied transactions.

File: docs/FAQ.md [hunk 2]
```diff
@@ -6,4 +6,40 @@ description: Frequently Asked Questions
...
+
+## How can I make importing of accounts faster?
+When calling [`import_account`](https://mobilecoin.gitbook.io/full-service-api/api-endpoints/v2/account/account/import_account), scanning generally starts at the Origin block (#1).
+By using the **first_block_index** param, a user can specify the starting block for that entropy if it is known. Newly created accounts are given a starting block index. 
+
+
+
+
+## How can I tell if FS is able to properly communicate with the Mobilecoin blockchain?
+Using the [`get_network_status`](https://mobilecoin.gitbook.io/full-service-api/api-endpoints/v2/network-status/get_network_status) API is the cleanest way to determine if FullService is successfully able to communicate with the blockchain.
```
Comment (brucek) [inline docs/FAQ.md:21]: Full Service is 2 words.  Should probably always write it out completely in public docs like these.

File: docs/FAQ.md [hunk 3]
```diff
@@ -6,4 +6,40 @@ description: Frequently Asked Questions
...
+
+
+## How can I make importing of accounts faster?
+When calling [`import_account`](https://mobilecoin.gitbook.io/full-service-api/api-endpoints/v2/account/account/import_account), scanning generally starts at the Origin block (#1).
+By using the **first_block_index** param, a user can specify the starting block for that entropy if it is known. Newly created accounts are given a starting block index. 
+
+
+
+
+## How can I tell if FS is able to properly communicate with the Mobilecoin blockchain?
```
Comment (brucek) [inline docs/FAQ.md:20]: FS --> Full Service

File: docs/FAQ.md [hunk 4]
```diff
@@ -6,4 +6,40 @@ description: Frequently Asked Questions
...
+Using the [`get_network_status`](https://mobilecoin.gitbook.io/full-service-api/api-endpoints/v2/network-status/get_network_status) API is the cleanest way to determine if FullService is successfully able to communicate with the blockchain.
+For instance, if your version of FS can't decipher blocks after a mobilecoin-core update, you may see something like this:
```
Comment (brucek) [inline docs/FAQ.md:22]: FS --> Full Service

File: docs/SUMMARY.md [hunk 5]
```diff
@@ -143,3 +143,4 @@
   * [Example Usage](../mirror/EXAMPLE.md)
 
 ## Frequently Asked Questions
+* [What is the Precision of Mob?](FAQ.md#what-is-the-precision-of-mob)
```
Comment (brucek) [inline docs/SUMMARY.md:146]: I feel like MOB should be capitalized?

**Review by brucek — CHANGES_REQUESTED** [review]
Some minor nits but this looks generally good otherwise

**Review by trunkafunk — COMMENTED** [review]
Comments addressed

**Review by briancorbin — APPROVED** [review]

**Review by brucek — APPROVED** [review]

**Pull request merged**